src/content/reference/react/useContext.md

```diff
@@ -195,13 +195,14 @@ function MyPage() {
 }
 ```
 
-Now any `Button` inside of the provider will receive the current `theme` value. If you call `setTheme` to update the `theme` value that you pass to the provider, all `Button` components will re-render with the new `'light'` value.
+Sekarang setiap `Button` di dalam penyedia akan menerima nilai `theme` saat ini. Jika Anda memanggil `setTheme` untuk memperbarui nilai `theme` yang Anda berikan ke penyedia, semua komponen `Button` akan dirender ulang dengan nilai `'light' yang baru.
 
 <Recipes titleText="Examples of updating context" titleId="examples-basic">
 
-#### Updating a value via context {/*updating-a-value-via-context*/}
+#### Memperbarui nilai melalui konteks {/*updating-a-value-via-context*/}
+
+Dalam contoh ini, komponen `MyApp` menyimpan variabel status yang kemudian diteruskan ke penyedia `ThemeContext`. Mencentang kotak centang "Dark mode" akan memperbarui *state*. Mengubah nilai yang disediakan akan merender ulang semua komponen yang menggunakan konteks tersebut.
```

Comment:
Iya betul soalnya semakin banyak PR yang ada kata tersebut. Di grup chat internal juga perlu dikonfirmasikan. Dan nanti perlu dijelaskan di Wiki/Glosarium.

- Kalau kaidah bahasa yang berlaku di bahasa Indonesia: `me-render`
- Kalau tanpa kaidah tapi lebih sederhana: `merender`

Reply:
Let's use kaidah KBBI to enforce conistency in the long term. The further we deviate from KBBI standards, the more manual/incidental adjustments we need to do.

Reply:
@zainfathoni Agree with this. The remaining problems are:

- Have to make sure everybody else understands this. And actually, I saw the majority of PRs started as following *kaidah* KBBI. 
- What about the merged PRs? Most of those were not using *kaidah* KBBI.

Reply:
Previously I just follow @resir014's decision.
I think it already discussed 😂
If we decide to use `-` separator, we also need to make change in the previous PRs

Reply:
So far, I have count of word "dirender", "di-render", "me-render", and "merender".

* `dirender` : `3`
* `di-render` : `8`
* `merender` : `1`
* `me-render` : `12`

> I counted based on commit 055079b

Reply:
> @mazipan If we decide to use `-` separator, we also need to make changes in the previous PRs

I think we can do the clean-up (spring cleaning) later on after most of the pages have been translated. It will be as simple as find and replace.

> @r17x So far, I counted word "dirender", "di-render", "me-render", and "merender".

Mostly separated with `-`

Yes, that's also why, in most existing PRS and upcoming PRs, we and they are mostly following the *kaidah*.

Reply:
Let's leave it be for now then. We can still clean up again later.